official/vision/beta/serving/run_lib.py

```diff
@@ -133,3 +138,97 @@ def get_export_module(experiment: str,
         type(params.task)))
 
   return export_module
+
+
+def read_class_names(class_names_path):
+  """Reads class names from text file.
+  Supports .txt and .json.
+  
+  Args:
+    class_names_path: `str`, path to json/txt file containing classes. 
+      Text file should contain one class name per line.
+      Json file should contain only one dictionary, `Mapping[int, str]`
+  """
+  
+  names = {}
+  if class_names_path.endswith('.txt'):
+    with open(class_names_path, 'r') as data:
+      for idx, name in enumerate(data):
+        names[idx] = name.strip('\n')
+  
+  elif class_names_path.endswith('.json'):
+    with open(class_names_path) as f:
+      names = json.load(f)
+    if type(list(names.keys())[0]) == str and type(list(names.values())[0]) == int:
+      names = dict((v,k) for k,v in names.items())
+
+  else:
+    raise NotImplementedError('File type is not .txt or .json, path %s' %class_names_path)
+
+  assert type(list(names.keys())[0]) == int, 'Loaded dict %s has wrong key type %s' %(
```

Comment:
This is a minor thing but try not to use assertions for non testing code, if python is run with the optimized flags this assertions might be dropped. Maybe we can just raise the error 

https://stackoverflow.com/questions/40182944/whats-the-difference-between-raise-try-and-assert